<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usermanager</class>
 <widget class="QWidget" name="usermanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>usermanager</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>421</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">backgorund-color:#f2f2f0;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:blod;</string>
    </property>
    <property name="text">
     <string>查询用户</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>731</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity:0.7;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#fff;
border-radius:5;
border: 1px solid #dbdbd9;
padding-left:10;
color:#555;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff3737;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>该用户不存在</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">width:50;
height:24;
background-color:#ffbb07;
color:#c97002;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#666;</string>
    </property>
    <property name="text">
     <string>111111111111111111111111</string>
    </property>
    <property name="buddy">
     <cstring>label_11</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">width:50;
height:24;
background-color:#ffbb07;
color:#c97002;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">width:50;
height:24;
background-color:#ffbb07;
color:#c97002;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>删除用户</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>确认密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>新密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff3737;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>原密码不正确</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_25">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#fff;
border-radius:5;
border: 1px solid #dbdbd9;
padding-left:10;
color:#555;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff3737;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>请输入中文+英文</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#717170;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>原密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff3737;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>两次输入不一致</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#fff;
border-radius:5;
border: 1px solid #dbdbd9;
padding-left:10;
color:#555;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_28">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#fff;
border-radius:5;
border: 1px solid #dbdbd9;
padding-left:10;
color:#555;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_30">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">width:50;
height:24;
background-color:#ffbb07;
color:#c97002;
border-radius:10;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_31">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">width:50;
height:24;
background-color:#ffbb07;
color:#c97002;
border-radius:10;
font-weight:normal;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#666;</string>
    </property>
    <property name="text">
     <string>1111111111111111111111</string>
    </property>
    <property name="buddy">
     <cstring>label_11</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#666;</string>
    </property>
    <property name="text">
     <string>1111111111111111111111</string>
    </property>
    <property name="buddy">
     <cstring>label_11</cstring>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_14</tabstop>
  <tabstop>pushButton_15</tabstop>
  <tabstop>lineEdit_22</tabstop>
  <tabstop>lineEdit_25</tabstop>
  <tabstop>lineEdit_28</tabstop>
  <tabstop>pushButton_30</tabstop>
  <tabstop>pushButton_31</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
